assemblies/assembly-ba-artificial-intelligence.adoc

```diff
@@ -0,0 +1,57 @@
+[id='assembly-ba-artificial-intelligence']
+= Integrating artificial intelligence with {PRODUCT}
+ifdef::context[:parent-context: {context}]
+
+:context: artificial-intelligence
+
+= What is artificial intelligence?
+When you think about artificial intelligence (AI), machine learning and big data might come to mind. But machine learning is only part of the picture. Artificial intelligence encompasses the following technologies:
+* Robotics: The integration of technology, science, and engineering that produces machines that can perform physical tasks that are performed by humans
+* Machine learning: The ability of a collection of algorithms to learn or improve when exposed to data without being explicitly programmed to do so
+* Natural language processing: A subset of machine learning that processes human speech
+* Mathematical optimization: The use of conditions and constraints to find the optimal solution to a problem
+* Digital decisioning: The use of defined criteria, conditions, and a series of machine and human tasks to make decisions
+
+While science fiction is filled with what is referred to as pure AI, machines that perform better than people and cannot be distinguished from them and learn and evolve without human intervention or control, pure AI is decades away. Meanwhile, we have pragmatic AI which is a lot less frightening and much more useful to us today. Pragmatic AI is a collection of AI technologies that combined provide solutions to problems such as predicting customer behavior, providing automated customer service, and helping customers make purchasing decisions.
```

Comment:
I acknowledge the PPT reported "Pure AI", but the correct term we should use here is

> Artificial general intelligence

ref: https://en.wikipedia.org/wiki/Artificial_general_intelligence

Reply:
(the "Pragmatic AI" term is fine)